Zend/tests/closure_compare.phpt

```diff
@@ -0,0 +1,74 @@
+--TEST--
+Closure comparison
+--FILE--
+<?php
+function foo() {
+    static $var;
+}
+
+$closures[0] = Closure::fromCallable('foo');
+$closures[1] = Closure::fromCallable('foo');
+
+printf("foo == foo: %s\n", $closures[0] == $closures[1] ? "OK" : "FAIL");
+
+$closures[0] = Closure::fromCallable('strlen');
+$closures[1] = Closure::fromCallable('strlen');
+
+printf("strlen == strlen: %s\n", $closures[0] == $closures[1] ? "OK" : "FAIL");
+
+$closures[0] = Closure::fromCallable('strlen');
+$closures[1] = Closure::fromCallable('strrev');
+
+printf("strlen != strrev: %s\n", $closures[0] != $closures[1] ? "OK" : "FAIL");
+
+class Foo {
+    public function __call($method, $args) {
+    
+    }
+
+    public function exists() {}
+}
+
+class Bar extends Foo {}
+
+$foo = new Foo;
+
+$closures[0] = Closure::fromCallable([$foo, "exists"]);
+$closures[1] = Closure::fromCallable([$foo, "exists"]);
+
+printf("foo::exists == foo::exists: %s\n", $closures[0] == $closures[1] ? "OK" : "FAIL");
```

Comment:
What about adding the test: "foo::exists != bar::exists"? I guess it must be considered as a different context. Same if the method was `static`, right?